README.md

```diff
@@ -392,6 +392,81 @@ spec:
 
 The container `app` does not contain CPU limits specified, which fails the overall evaluation as shown in the screenshot.
 
+#### Using Input Parameters
+
+Guard allows you to use input parameters for dynamic data lookups during validation. This feature is particularly useful when you need to reference external data in your rules.
```

Comment:
Do we want to specify how guard doesn't allow keys input params to have conflicting paths?

Reply:
Yup - will add this!